manifests/quargoutfitter.yaml

```diff
@@ -0,0 +1,15 @@
+name: Quarg Outfitter
+authors: Nova.
+homepage: https://github.com/Nova1422/plugins?tab=readme-ov-file#quargoutfitter
+license: ALL-RIGHTS-RESERVED
+version: v1.0.0-quargoutfitter
+shortDescription: Allows Purchase of Quarg Outfits on specific planets. The Planets are, "Smuggler's Den", and "Ring of Power"/"Remote Blue", Depending on which side you chose in coalition space Also removes Quarg Death Sentence.
```

Comment:
this shortDescription is too long (should be less than 200c, must be less than 150c)